### PR TITLE
Doc tweak: standalone components usage instructions

### PR DESCRIPTION
**Pull request by 0xfraso**
Hi there,

just opened this PR to help people find instructions about using `ngx-translate` in standalone based applications

hope you find this useful

### CODE REVIEW

**Review by carlosdag28 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -101,6 +102,32 @@ you can export the `TranslateModule` to make sure you don't have to import it in
...
+    providers: [
+        ...
+        importProvidersFrom(HttpClientModule), // or provideHttpClient() in Angular v15+
+        importProvidersFrom(TranslateModule.forRoot({
+            loader: {
+                provide: TranslateLoader,
+                useFactory: createTranslateLoader,
+                deps: [HttpClient]
+           }
+        }),
```
Comment (oxydeon) [inline README.md:120]: missing `)` here:

`        })),`

Comment (0xfraso) [reply README.md]: I'm sorry for the typo, just pushed a quick fix

**Review by oxydeon — APPROVED** [review]

**Comment by markodjunev** [comment]
I did exactly the same thing. However I got the following error **TypeError: this.handler.handle is not a function** on  _translate.use()_. This is my code:

@Component({
  selector: 'app-header',
  standalone: true,
  imports: [CommonModule, TranslateModule],
  templateUrl: './header.component.html',
  styleUrl: './header.component.css'
})
export class HeaderComponent {
  private languages: string[] = ['fr', 'en'];

  constructor(
    protected translate: TranslateService
  ) {
    translate.addLangs(this.languages);

    translate.use('en')
  }

**Comment by 0xfraso** [comment]
Hi @markodjunev, by default ngx-translate uses `HttpClient` as dependency, so make sure to put `provideHttpClient()` as provider into your bootstrap function.

**Comment by CodeAndWeb** [comment]
I've included it in the docs... 
https://ngx-translate.org/getting-started/translation-files/#standalone-components

**Pull request closed without merging**